apps/ocp-plugin/start-ocp-console.sh

```
@@ -19,15 +19,18 @@ echo "Starting local OpenShift console..."
+# The console's internal mechanism for user authentication via OAuth is disabled.
+# We are providing the authentication token directly to the console process, which it will use for all API calls.
+BRIDGE_USER_AUTH="disabled"
 BRIDGE_K8S_MODE="off-cluster"
-BRIDGE_K8S_AUTH="bearer-token"
 BRIDGE_CA_FILE="/tmp/ca.crt"
 BRIDGE_USER_AUTH_OIDC_CLIENT_ID="console-oauth-client"
 BRIDGE_USER_AUTH_OIDC_CLIENT_SECRET_FILE="/tmp/console-client-secret"
 BRIDGE_USER_AUTH_OIDC_CA_FILE="/tmp/ca.crt"
 BRIDGE_K8S_MODE_OFF_CLUSTER_SKIP_VERIFY_TLS=true
 BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=$(oc whoami --show-server)
+BRIDGE_K8S_AUTH_BEARER_TOKEN=$(oc whoami --show-token 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] BRIDGE_K8S_AUTH_BEARER_TOKEN appears unused. Verify use (or export if used externally).

(SC2034)